src/test/enums.spec.js

```diff
@@ -64,73 +64,3 @@ function go(s : MyStructXXX, t: EnumTypedefXXX) {
     }
   )
 );
-
-test(
```

Comment:
Can you add some tests for the new behavior?

Reply:
There is a test above this one that I believe covers this use case. I can double check though